sources/whippet/src/policy.rs

```diff
@@ -920,4 +1033,58 @@ mod tests {
         clean_connect_rules(4, &mut rules);
         assert!(rules.is_empty());
     }
+
+    #[test_case(Rule::Send {
+            allow: false,
+            priority: u64::default(),
+            send_broadcast: u32::default(),
+            send_destination: "*".to_string(),
+            send_interface: "*".to_string(),
+            send_member: "*".to_string(),
+            send_path: "*".to_string(),
+            send_type: MessageType::default(),
+        }; "with a send rule wildcards are replaced with empty strings")]
+    #[test_case(Rule::Receive {
+            allow: false,
+            priority: u64::default(),
+            receive_broadcast: u32::default(),
+            receive_interface: "*".to_string(),
+            receive_member: "*".to_string(),
+            receive_path: "*".to_string(),
+            receive_sender: "*".to_string(),
+            receive_type: MessageType::default(),
+        }; "with a receive rule wildcards are replaced with empty strings")]
+    fn rules_with_wildcards(rule: Rule) {
+        assert_eq!(rule.name().unwrap(), "");
+        assert_eq!(rule.interface().unwrap(), "");
+        assert_eq!(rule.member().unwrap(), "");
+        assert_eq!(rule.path().unwrap(), "");
+    }
+
+    #[test_case(Rule::Send {
+            allow: false,
+            priority: u64::default(),
+            send_broadcast: u32::default(),
+            send_destination: "name".to_string(),
+            send_interface: "interface".to_string(),
+            send_member: "member".to_string(),
+            send_path: "path".to_string(),
+            send_type: MessageType::default(),
```

Comment:
If you don't care about the default values and aren't asserting them, does this work? Same with the other test cases.
```suggestion
            send_destination: "name".to_string(),
            send_interface: "interface".to_string(),
            send_member: "member".to_string(),
            send_path: "path".to_string(),
            ..Default::default()
```

Reply:
~~I could do that, but I have to add `#[default]` to `Rule` and since it isn't a unit `enum` I have to manually implement the `Default` trait for it.~~

Correction on this, I did try this by implementing `Default` like so:

```rust
impl Default for Rule {
    fn default() -> Self {
        Self::Send {
            send_destination: Default::default(),
            send_interface: Default::default(),
            send_member: Default::default(),
            send_type: Default::default(),
            send_path: Default::default(),
            send_broadcast: Default::default(),
            allow: Default::default(),
            priority: Default::default(),
        }
    }
}
```

But got this error:

```bash
┌───────────────────> 🌐 ip-172-31-52-240 in ~/Code/work/by-feature/whippet/bottlerocket-core-kit/sources/whippet
└─> on  fix-whippet-defaults [$!] is 📦 v0.1.0 via 🦀 v1.88.0 ✦ ❯ cargo test
   Compiling whippet v0.1.0 (/home/agarrcia/Code/work/by-feature/whippet/bottlerocket-core-kit/sources/whippet)
error[E0436]: functional record update syntax requires a struct
    --> whippet/src/policy.rs:1057:15
     |
1057 |             ..Default::default()
     |               ^^^^^^^^^^^^^^^^^^

For more information about this error, try `rustc --explain E0436`.
error: could not compile `whippet` (bin "whippet" test) due to 1 previous error
┌───────────────────> 🌐 ip-172-31-52-240 in ~/Code/work/by-feature/whippet/bottlerocket-core-kit/sources/whippet
└─> on  fix-whippet-defaults [$!] is 📦 v0.1.0 via 🦀 v1.88.0 ✦ ❯
```

Reply:
Doesn't seem worth it then unless you're excited about that!